Tests/Tests.swift

```diff
@@ -141,7 +141,7 @@ class Tests: XCTestCase {
 
         let nsRange = string.range(of: expected)
         guard let range = nsRange.range(on: string) else {
-            XCTAssertFalse(true)
+            XCTAssertTrue(false, "Could not get range of substring.")
```

Comment:
You could just use `XCTFail` with this message and the others

Reply:
That's the thing I was looking for, but couldn't remember! 👍